18F/identity-idp
# Update Rails to 7.1.3.3

## 🛠 Summary of changes

Updates Rails from 7.1.3.2 to 7.1.3.3

See release post: https://rubyonrails.org/2024/5/17/Rails-Versions-7-0-8-2-and-7-1-3-3-have-been-released

This resolves a security advisory for ActionText on `main`.

## 📜 Testing Plan

Verify the application runs without issue:

1. Go to http://localhost:3000

Validate that security advisories are resolved:

1. Run `bundle exec bundler-audit check --update`
